src/more/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import directive from './directive';
+
+const moduleName = 'ngOvhResponsiveTabsMore';
+
+angular
+  .module(moduleName, [])
+  .directive('responsiveTabMore', directive);
```

Comment:
What's the purpose of creating a new module that contain the `responsiveTabMore` directive?

Why not adding that directive to the `ngOvhResponsiveTabs` module?

Reply:
Sure, modules `tab` & `more` deleted.